### PR TITLE
Updating git workflows to use node 14

### PR DESCRIPTION
**Pull request by ribeiroguilherme**
## Summary
- Updating workflows to use node 14

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x]
```
Comment (sponglord) [inline .github/workflows/ci-test.yml:15]: Is their any reason/advantage to know that it still builds in Node 12?

**Comment by m1aw** [comment]
Should we also make run on the current LTS (16.x.x)?

File: .github/workflows/ci-test.yml (hunk 1 above)
Comment (ribeiroguilherme) [reply .github/workflows/ci-test.yml:15]: Not really as far as I know - this line was added 2 years ago where Node 12 had [active support and security support](https://endoflife.date/nodejs). I believe that is just legacy config.

**Comment by ribeiroguilherme** [comment]
> Should we also make run on the current LTS (16.x.x)?

Makes sense, as we are internally planning to upgrade to Node 16 any time soon. I will add it to the CI pipeline at least.

**Comment by ribeiroguilherme** [comment]
> > Should we also make run on the current LTS (16.x.x)?
> 
> Makes sense, as we are internally planning to upgrade to Node 16 any time soon. I will add it to the CI pipeline at least.

Ok it seems that adding Node16 breaks and the tests don't pass - we need to fix few of them that are throwing `UnhandledPromiseRejection` . Better to do this in another PR, and keep the scope of this one to upgrade to Node14.

**Review by m1aw — APPROVED** [review]

**Pull request merged**